test/services/submission/EntitiesBlockBuilderSpec.scala

```diff
@@ -117,6 +117,8 @@ class EntitiesBlockBuilderSpec extends VatRegSpec with MockPartnersService with
       }
       "an SA UTR was not provided" should {
         "return a JSON array containing a single partner without identifiers" in {
+          val testEntity = testSoleTraderEntity.copy(bpSafeId = Some(testBpSafeId), sautr = None)
+          val testPartner = Partner(details = testEntity, partyType = Individual, isLeadPartner = true)
           mockGetPartners(testRegId)(Future.successful(Some(List(testPartner.copy(details = testEntityNoSafeId.copy(sautr = None))))))
```

Comment:
change to
          val testEntity = testSoleTraderEntity.copy(bpSafeId = None, sautr = None)
          val testPartner = Partner(details = testEntity, partyType = Individual, isLeadPartner = true)
          mockGetPartners(testRegId)(Future.successful(Some(List(testPartner))))